tests/test_meshtastic_utils.py

```
@@ -248,6 +254,68 @@ def _done_future(*args, **kwargs):
+
+        with (
+            patch("mmrelay.meshtastic_utils.time.time", return_value=1_000_010.0),
+            patch(
+                "mmrelay.meshtastic_utils._is_health_probe_response_packet",
+                return_value=True,
+            ) as mock_health_probe,
+        ):
+            on_meshtastic_message(packet, MagicMock())
+
+        mock_health_probe.assert_called_once()
+        self.assertAlmostEqual(mu._relay_rx_time_clock_skew_secs or 0.0, 5_000.0)
```

Ruff (0.15.7)

[warning] Use a regular `assert` instead of unittest-style `assertAlmostEqual`

(PT009)